### PR TITLE
Audio Track AutoDispose Property

### PR DESCRIPTION
**Pull request by staravia**
This property will determine whether an audio track will be disposed automatically

### CODE REVIEW

File: Wobble/Audio/Tracks/AudioTrack.cs [hunk 1]
```diff
@@ -140,9 +145,11 @@ public double Volume
         /// </summary>
         /// <param name="path"></param>
         /// <param name="preview"></param>
-        public AudioTrack(string path, bool preview = false)
+        /// <param name="dontAutoDispose></param>
```
Comment (YaLTeR) [inline Wobble/Audio/Tracks/AudioTrack.cs]: You're missing a closing `"` which breaks syntax highlighting.

File: Wobble/Audio/Tracks/AudioTrack.cs [hunk 2]
```diff
@@ -140,9 +145,11 @@ public double Volume
...
+        public AudioTrack(string path, bool preview = false, bool dontAutoDispose = false)
```
Comment (YaLTeR) [inline Wobble/Audio/Tracks/AudioTrack.cs]: `bool autoDispose = true`?

File: Wobble/Audio/Tracks/AudioTrack.cs [hunk 3]
```diff
@@ -155,9 +162,11 @@ public AudioTrack(string path, bool preview = false)
         /// </summary>
         /// <param name="data"></param>
         /// <param name="preview"></param>
-        public AudioTrack(byte[] data, bool preview = false)
+        /// <param name="dontAutoDispose></param>
```
Comment (YaLTeR) [inline Wobble/Audio/Tracks/AudioTrack.cs]: Missing `"`.

File: Wobble/Audio/Tracks/AudioTrack.cs [hunk 4]
```diff
@@ -170,9 +179,11 @@ public AudioTrack(byte[] data, bool preview = false)
         /// </summary>
         /// <param name="data"></param>
         /// <param name="preview"></param>
-        public AudioTrack(Stream data, bool preview = false)
+        /// <param name="dontAutoDispose></param>
```
Comment (YaLTeR) [inline Wobble/Audio/Tracks/AudioTrack.cs]: Missing `"`.

File: Wobble/Audio/Tracks/AudioTrack.cs [hunk 5]
```diff
@@ -155,9 +162,11 @@ public AudioTrack(string path, bool preview = false)
         /// </summary>
         /// <param name="data"></param>
         /// <param name="preview"></param>
-        public AudioTrack(byte[] data, bool preview = false)
+        /// <param name="dontAutoDispose></param>
+        public AudioTrack(byte[] data, bool preview = false, bool dontAutoDispose = false)
```
Comment (YaLTeR) [inline Wobble/Audio/Tracks/AudioTrack.cs]: `bool autoDispose = true`

File: Wobble/Audio/Tracks/AudioTrack.cs [hunk 6]
```diff
@@ -170,9 +179,11 @@ public AudioTrack(byte[] data, bool preview = false)
         /// </summary>
         /// <param name="data"></param>
         /// <param name="preview"></param>
-        public AudioTrack(Stream data, bool preview = false)
+        /// <param name="dontAutoDispose></param>
+        public AudioTrack(Stream data, bool preview = false, bool dontAutoDispose = false)
```
Comment (YaLTeR) [inline Wobble/Audio/Tracks/AudioTrack.cs]: `bool autoDispose = true`

File: Wobble/Audio/Tracks/AudioTrack.cs (hunk 2 above)
Comment (staravia) [reply Wobble/Audio/Tracks/AudioTrack.cs]: not a fan of that, i'll ask swan

Comment (staravia) [reply Wobble/Audio/Tracks/AudioTrack.cs]: i decided to stick to your suggestion

**Pull request merged**